app/extensions/brave/locales/en-US/app.properties

```diff
@@ -207,9 +207,9 @@ runInsecureContentWarning=This page is trying to load scripts from insecure sour
 runtimeUpdatePending=A runtime update which requires the application to be restarted is available.
 safebrowsingError=Dangerous Site Blocked
 safebrowsingErrorAdvanced=Advanced
-safebrowsingErrorBypass=You can access this site by clicking the Brave icon in the top right corner and temporarily disabling 'Block Phishing / Malware'. This is not recommended!
+safebrowsingErrorBypass=You can access this site by navigating to 'about:preferences#shields' and disabling 'Block Phishing/Malware'. This is not recommended!
 safebrowsingErrorHideAdvanced=Hide advanced
-safebrowsingErrorInfo=Brave uses crowdsourced blacklists from Spam404 and Disconnect.me to block dangerous domains. If you believe this domain should not be blocked, Please report the site at https://community.brave.com/c/site-blocking
+safebrowsingErrorInfo=Brave uses crowdsourced blacklists from Google Safe Browsing, Spam404 and Disconnect.me to block dangerous domains. If you believe this domain should not be blocked, Please report the site at https://community.brave.com/c/site-blocking
```

Comment:
Does this text show up if Google safebrowsing is enabled, or only if our own safebrowsing implementation is enabled?

Reply:
This text will only show up if `Brave Safe Browsing` is enabled.